# /database/srv/SetPotActive.srv

uint32 id
bool active
---
bool success